sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -328,6 +335,24 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     return offeringQuery.getOne();
   }
 
+  /**
+   * Updates basic offering data that does not affect the assessment
+   * and does not require the complete offering validation.
+   * @param basicOffering information to be updated.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @returns Education program offering created.
+   */
+  async updateEducationProgramOfferingBasicData(
+    offeringId: number,
+    basicOffering: EducationProgramOfferingBasicData,
```

Comment:
IMO the property name can be offeringBasicData or offeringBasicPayload instead of basicOffering

Reply:
I will rename it to `basicOfferingData`. Hope that it is ok.